### PR TITLE
Make some changes to reviewdb ui and add badges to it

### PR DESCRIPTION
**Pull request by mantikafasi**
merge or explode

### CODE REVIEW

**Comment by KraXen72** [comment]
move the reviewdb to userProfile pls thanks

File: src/plugins/reviewDB/components/ReviewBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            {({ onMouseEnter, onMouseLeave }) => (
+                <img
+                    width="22px"
+                    height="22px"
+                    onMouseEnter={onMouseEnter}
+                    onMouseLeave={onMouseLeave}
+                    src={badge.badge_icon}
+                    style={{ verticalAlign: "middle", marginLeft: "4px" }}
+                    onClick={() =>
+                        VencordNative.ipc.invoke(IpcEvents.OPEN_EXTERNAL, badge.redirect_url)
```
Comment (Vendicated) [inline src/plugins/reviewDB/components/ReviewBadge.tsx:36]: please use discord's url opener instead that asks for consent

File: src/plugins/reviewDB/entities/Badge.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+/* eslint-disable semi */
```
Comment (Vendicated) [inline src/plugins/reviewDB/entities/Badge.ts:19]:
```suggestion
```

Comment (mantikafasi) [reply src/plugins/reviewDB/entities/Badge.ts:19]: eslint shits itself if I remove it

Comment (Vendicated) [reply src/plugins/reviewDB/entities/Badge.ts:19]: then fix it lmao

Comment (mantikafasi) [reply src/plugins/reviewDB/entities/Badge.ts:19]: Tell me how

Comment (KraXen72) [reply src/plugins/reviewDB/entities/Badge.ts:19]: run auto fix on it or add semicolons manually

Comment (mantikafasi) [reply src/plugins/reviewDB/entities/Badge.ts:19]: Then it tells me to remove semicolons

File: src/plugins/reviewDB/components/ReviewBadge.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import { Badge } from "../entities/Badge";
+
+export default function ReviewBadge(badge: Badge) {
+    return (
+        <Tooltip
+            text={badge.badge_name}>
+            {({ onMouseEnter, onMouseLeave }) => (
+                <img
+                    width="22px"
```
Comment (Vendicated) [inline src/plugins/reviewDB/components/ReviewBadge.tsx:29]: 22? Should probably be 24 to be a nice number and stay consistent with Discord

File: src/plugins/reviewDB/components/ReviewBadge.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                    height="22px"
+                    onMouseEnter={onMouseEnter}
+                    onMouseLeave={onMouseLeave}
+                    src={badge.badge_icon}
```
Comment (Vendicated) [inline src/plugins/reviewDB/components/ReviewBadge.tsx:33]: it would be good to add some alt text

**Review by Vendicated — APPROVED** [review]
looks good other than the above reviews, please fix conflict!

**Comment by KraXen72** [comment]
by the time this pr gets merged my email will no longer accept any new emails because i once commented something here and manti is still commiting some shit

**Comment by mantikafasi** [comment]
love

**Pull request merged**